### DISCUSSION TITLE
how about support debug javascript in html

### DISCUSSION THREAD

**Discussion by allenliuvip** [LazyVim/LazyVim#1021] [Q&A]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

I want  to debug javascript in html

### Describe the solution you'd like

bulabulabula

### Describe alternatives you've considered

bulabulabula

### Additional context

bulabulabula

**Comment by azdanov** [comment]
LazyVim uses [nvim-dap](https://github.com/mfussenegger/nvim-dap) as the debugger. You should ask there if debugging javascript in html is supported, and which adapter to use.

**Comment by folke** [comment]
Converted this to a discussion. Some people might share their config to get what you need, or check the nvim-dap docs yourself to see how to set it up as @azdanov mentioned
